packages/esm-patient-registration-app/src/patient-registration/input/basic-input/input/input.test.tsx

```diff
@@ -4,36 +4,6 @@ import userEvent from '@testing-library/user-event';
 import { Formik, Form } from 'formik';
 import { Input } from './input.component';
 
-describe.skip('number input', () => {
```

Comment:
I think it is okay to keep these tests. Are these test cases causing any errors or warnings?

Reply:
These are not the expected behaviour of the input component. input component is not getting used as any of the following.